static/js/containers/pages/CheckoutPage.js

```diff
@@ -1,5 +1,8 @@
 // @flow
+/* global SETTINGS: false */
 import React from "react"
+import DocumentTitle from "react-document-title"
+import * as Constants from "../../constants"
```

Comment:
maybe just do `import CHECKOUT_PAGE_TITLE from "../../constants"` since it is the only one used on this page.  Same for the other pages.

Reply:
Updated.